qiskit_experiments/library/calibration/analysis/drag_analysis.py

```diff
@@ -28,32 +28,37 @@
 class DragCalAnalysis(CurveAnalysis):
```

Comment:
Please add `Experiment Options` under `_default_experiment_options` method like this

https://github.com/Qiskit/qiskit-experiments/blob/d777235c94a5050ac8c2723006a5c21430573063/docs/_ext/custom_styles/example/example_experiment.py#L192-L195

Reply:
Done here: 7facc08